src/table/control_codes.h

```diff
@@ -65,6 +65,12 @@ enum StringControlCode {
 	SCC_VELOCITY,
 	SCC_HEIGHT,
 
+	SCC_UNITS_DAYS_OR_SECONDS,
+	SCC_UNITS_MONTHS_OR_MINUTES,
+	SCC_UNITS_YEARS_OR_PERIODS,
```

Comment:
There is only one usage of this, where the other subsidy related things use the next one. Do we actually want to have this one?

Reply:
I am not arguing the syntax I am arguing the usage :D Just to check if we say the same:

```
STR_CONFIG_SETTING_SUBSIDY_DURATION_VALUE                       :{NUM} year{P "" s}
STR_CONFIG_SETTING_SUBSIDY_DURATION_VALUE                       :{UNITS_YEARS_OR_PERIODS}

STR_NEWS_SERVICE_SUBSIDY_AWARDED_QUADRUPLE                      :{BIG_FONT}{BLACK}Service subsidy awarded to {RAW_STRING}!{}{}{STRING} from {STRING2} to {STRING2} will pay quadruple rates for the next {NUM} year{P "" s}!
STR_NEWS_SERVICE_SUBSIDY_AWARDED_QUADRUPLE                      :{BIG_FONT}{BLACK}Service subsidy awarded to {RAW_STRING}!{}{}{STRING} from {STRING2} to {STRING2} will pay quadruple rates for the next {UNITS_YEARS_OR_MINUTES}!
```

What I am arguing, is why not have them both `UNITS_YEARS_OR_MINUTES`. So you see in both cases N minutes, with wallclock, instead one N minutes, and the other M periods?

It just feels a bit weird, that when you see a subsidy you get: for the next 12 minutes.
But the setting is: 1 period

Why not make your live easier, and do both with "12 minutes", and remove this used-only-once SCC?

So just to be clear, I am talking `period -> 12 minutes` here, as you do the same with the rest of the subsidies. I am not arguing removing this SCC to replace it with some other complex stuff to have the same end-result :)

Reply:
![mismatch](https://github.com/OpenTTD/OpenTTD/assets/55058389/1204158d-cc4c-43d7-b0e3-b36b954e6cd3)

We discussed this on Discord, but I'll leave this here for future readers of this PR.

Unit conversion happens in the string control code, so this works, but the setting is still stored in years. 😉